### ISSUE TITLE
Setup , Data Loading

### ISSUE THREAD

**Issue by j-harshith-reddy** [Jason-cse/ML-wine_quality#22]
import warnings
warnings.filterwarnings('ignore')

import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

from sklearn.compose import ColumnTransformer
from sklearn.pipeline import Pipeline
from sklearn.impute import SimpleImputer
from sklearn.preprocessing import StandardScaler, OneHotEncoder
from sklearn.model_selection import train_test_split, StratifiedKFold, cross_val_score, GridSearchCV

from sklearn.linear_model import LogisticRegression
from sklearn.neighbors import KNeighborsClassifier
from sklearn.naive_bayes import GaussianNB
from sklearn.tree import DecisionTreeClassifier
from sklearn.svm import SVC
from sklearn.ensemble import RandomForestClassifier, AdaBoostClassifier, GradientBoostingClassifier, BaggingClassifier
from sklearn.neural_network import MLPClassifier

from sklearn.metrics import (accuracy_score, precision_score, recall_score, f1_score,
                             confusion_matrix, classification_report, roc_auc_score, roc_curve)

sns.set_style('whitegrid')
sns.set_palette('colorblind')
RANDOM_STATE = 42
np.random.seed(RANDOM_STATE)

df = pd.read_csv("wine-quality-white-and-red.csv")
print('Loaded dataset shape:', df.shape)
df.head(5)
